sui-execution/latest/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -619,6 +619,16 @@ impl<'vm, 'state, 'a> ExecutionContext<'vm, 'state, 'a> {
                                 ));
                             }
                         }
+                        Some(Value::Receiving(_, _)) => {
```

Comment:
Is it necessarily a problem not to receive the object?  This prevents patterns where you might conditionally receive an object.

Reply:
I don't believe there's an issue with not receiving in the transaction, but I will want to think through that some more...

Reply:
We now support this! You can either receive or not receive an object in a transaction :)  (I've added a bunch of tests in the transactional tests to make sure we support this).